projects/packages/image-cdn/src/class-image-cdn.php

```diff
@@ -651,7 +651,7 @@ public static function filter_the_content( $content ) {
 					if ( isset( $placeholder_src ) && self::validate_image_url( $placeholder_src ) ) {
 						$placeholder_src = Image_CDN_Core::cdn_url( $placeholder_src );
 
-						if ( $placeholder_src !== $placeholder_src_orig ) {
+						if ( isset( $placeholder_src_orig ) && $placeholder_src !== $placeholder_src_orig ) {
```

Comment:
Looks to me like there's a bug in this code, which we're not really fixing by adding this check.

I think the real fix would probably be to assign `$placeholder_src` and `$placeholder_src_orig` as null near line 366 so they get reset on each iteration of the loop, instead of possibly carrying over to later tags.

Then this `isset` won't be necessary, and for that matter line 651 could be changed to `$placeholder_src !== null`.

Reply:
eaca5653b0